core/util.py

```diff
@@ -127,6 +125,10 @@ def load_example_data():
         "risk_register",
     ]:
         table_df = pd.read_csv(resources / f"{table}.csv")
+        if table == "project_dim":
```

Comment:
can this be done much more efficiently with Pandas vectorised .str.split method? ie something like

`table_df["postcodes"] = table_df["postcodes"].str.split(",")`

Reply:
This not resolved, please switch to above format. It should work, I previously tried it locally and all tests still passed

Reply:
so replace 
```
for x in range(len(table_df.postcodes.values)):
                table_df.postcodes.values[x] = table_df.postcodes.values[x].split(",")
```
with
`table_df["postcodes"] = table_df["postcodes"].str.split(",")`

Reply:
this is much faster as it uses the dataframe.str vector method which applies string transformations as a vector to all rows, rather than the much less efficient iteration though all DF rows

Reply:
Hmm, just reading more about this, I think in this case iteration might be faster. The general consensus seems to be that whilst vectorised methods should usually be much faster, the string ones are not necessarily: https://stackoverflow.com/questions/54028199/are-for-loops-in-pandas-really-bad-when-should-i-care